spinalcordtoolbox/image.py

```diff
@@ -265,7 +265,7 @@ def __init__(self, param=None, hdr=None, orientation=None, absolutepath=None, di
         """
 
         # initialization of all parameters
-        self.im_file = None
```

Comment:
From a quick code search, it seems that the `im_file` attribute is indeed only used in these two files, and mostly just for its `im_file.affine` attribute.